src/robotka.h

```diff
@@ -591,6 +591,12 @@ void rkSmartLedsRGB(uint16_t idx, uint8_t r, uint8_t g, uint8_t b);
  */
 void rkSmartLedsHSV(uint16_t idx, uint8_t h, uint8_t s, uint8_t v);
 
+/**
+ *  \brief Nastavit barvu chytré led podle jejího HEX kódu 
```

Comment:
Mezera před `\brief` a napíš prosím, že je to RGB HEX kód - HEX je jen jiná reprezentace, barevný model je stále RGB.